lms/envs/test.py

```diff
@@ -1,562 +1,10 @@
-    '--xunit-file', _REPORT_DIR / 'nosetests.xml',
-]
-
-# Local Directories
-TEST_ROOT = path("test_root")
-# Want static files in the same dir for running on jenkins.
-STATIC_ROOT = TEST_ROOT / "staticfiles"
-
-STATUS_MESSAGE_PATH = TEST_ROOT / "status_message.json"
-
-COURSES_ROOT = TEST_ROOT / "data"
-DATA_DIR = COURSES_ROOT
-
-COMMON_TEST_DATA_ROOT = COMMON_ROOT / "test" / "data"
-# Where the content data is checked out.  This may not exist on jenkins.
-GITHUB_REPO_ROOT = ENV_ROOT / "data"
-
-USE_I18N = True
-LANGUAGE_CODE = 'en'  # tests assume they will get English.
-
-XQUEUE_INTERFACE = {
-    "url": "http://sandbox-xqueue.edx.org",
-    "django_auth": {
-        "username": "lms",
-        "password": "***REMOVED***"
-    },
-    "basic_auth": ('anant', 'agarwal'),
-}
-XQUEUE_WAITTIME_BETWEEN_REQUESTS = 5  # seconds
-
-# Don't rely on a real staff grading backend
-MOCK_STAFF_GRADING = True
-MOCK_PEER_GRADING = True
-
-############################ STATIC FILES #############################
-
-# TODO (cpennington): We need to figure out how envs/test.py can inject things
-# into common.py so that we don't have to repeat this sort of thing
-STATICFILES_DIRS = [
-    COMMON_ROOT / "static",
-    PROJECT_ROOT / "static",
-]
-STATICFILES_DIRS += [
-    (course_dir, COMMON_TEST_DATA_ROOT / course_dir)
-    for course_dir in os.listdir(COMMON_TEST_DATA_ROOT)
-    if os.path.isdir(COMMON_TEST_DATA_ROOT / course_dir)
-]
-
-# Avoid having to run collectstatic before the unit test suite
-# If we don't add these settings, then Django templates that can't
-# find pipelined assets will raise a ValueError.
-# http://stackoverflow.com/questions/12816941/unit-testing-with-django-pipeline
-STATICFILES_STORAGE = 'pipeline.storage.NonPackagingPipelineStorage'
-
-# Don't use compression during tests
-PIPELINE_JS_COMPRESSOR = None
-
-update_module_store_settings(
-    MODULESTORE,
-    module_store_options={
-        'fs_root': TEST_ROOT / "data",
-    },
-    xml_store_options={
-        'data_dir': mkdtemp_clean(dir=TEST_ROOT),  # never inadvertently load all the XML courses
-    },
-    doc_store_settings={
-        'host': MONGO_HOST,
-        'port': MONGO_PORT_NUM,
-        'db': 'test_xmodule',
-        'collection': 'test_modulestore{0}'.format(THIS_UUID),
-    },
-)
-
-CONTENTSTORE = {
-    'ENGINE': 'xmodule.contentstore.mongo.MongoContentStore',
-    'DOC_STORE_CONFIG': {
-        'host': MONGO_HOST,
-        'db': 'xcontent',
-        'port': MONGO_PORT_NUM,
-    }
-}
-
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': TEST_ROOT / 'db' / 'edx.db',
-        'ATOMIC_REQUESTS': True,
-    },
-
-}
 
 # This hack disables migrations during tests. We want to create tables directly from the models for speed.
 # See https://groups.google.com/d/msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ.
 MIGRATION_MODULES = {app: "app.migrations_not_used_in_tests" for app in INSTALLED_APPS}
```

Comment:
@doctoryes Any idea who added this line originally? I believe it was incorporated with the Django 1.8 changes. Should we be skipping migrations at all during CI testing?